index.md

```diff
@@ -8,8 +8,10 @@ less time, and with less pain. The lessons below were designed for
 those interested in working with social sciences data in R.
 
 This is an introduction to R designed for participants with no
-programming experience. These lessons can be taught in a day (~ 6
-hours). They start with some basic information about R syntax, the
+programming experience. These lessons can be taught in a half-day,
+full-day, or over a two-day workshop (see
+[Instructor Notes](https://datacarpentry.org/r-socialsci/instructor/instructor-notes.html)).
```

Comment:
The link to the instructor notes is put in twice, once in the markdown link and once outside it. Delete the second instance.